website/docs/guides/creating-pages.md

```diff
@@ -51,7 +51,7 @@ export default function Hello() {
 }
 ```
 
-Once you save the file, the development server will automatically reload the changes. Now open `http://localhost:3000/helloReact` and you will see the new page you just created.
+Once you save the file, the development server will automatically reload the changes. Now open [http://localhost:3000/helloReact](http://localhost:3000/helloReact) and you will see the new page you just created.
```

Comment:
I'm not sure if this is _necessarily_ useful—but I never followed this part of the tutorial when learning Docusaurus so I can't speak for others. Is it useful for you?

Reply:
I myself started following the tutorial half an hour ago and this is the first page I came across.
I found myself copy/pasting the links over and over so in the eyes of the next developer who will go through his/her first steps I personally found this change as useful.
Are you referring specifically to this markdown?

Reply:
Yes, to the Markdown files on the website. Do you read them with a dev server running and actually go through them step-by-step?

Reply:
Yes, as mentioned in the [fast track](https://docusaurus.io/docs#fast-track).
I installed a fresh site:
```sh
npx create-docusaurus@latest my-website classic
```

then started the site:
```sh
cd my-website
npx docusaurus start
```

and went on with the 3rd instruction:
> Open http://localhost:3000 and follow the tutorial.

Reply:
If the markdown you are referring to are not the one visible when running those steps I can revert those too, no problem. I may have made a mistake regarding which ones are served when you follow the guide. I simply search project wide the string: 
```
`http://locahost:3000
```

Reply:
OK, makes sense... Just not sure about further down where the instructions go into more details and often involves completely contrived examples that I assume people just don't bother copying.

Reply:
> If the markdown you are referring to is not the one visible when running those steps I can revert those too

Yeah those are further down, but I'm fine. If we are to do it we'd rather do it all so it's consistent. Maybe people will find it useful?

Reply:
I like the documentation so far, I find it to the point and easy to follow. You might say it's expected for a documentation-builder project :-)